nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/test/comment.test.js

```diff
@@ -197,12 +197,11 @@ describe("With OpenTelemetry tracing", () => {
 
     it('Starting an OpenTelemetry trace should produce `traceparent`', (done) => {
         const rootSpan = tracer.startSpan('rootSpan');
-
-        tracer.withSpan(rootSpan,  async () => {
+        context.with(trace.setSpan(context.active(), rootSpan), async () => {
```

Comment:
Can do this simpler with [`Tracer.startActiveSpan()`](https://open-telemetry.github.io/opentelemetry-js-api/interfaces/tracer.html#startactivespan)